app/helpers/related_urls_helper.rb

```diff
@@ -424,4 +424,10 @@ module RelatedUrlsHelper
     ['Simple Subset Wizard (SSW)', 'SIMPLE SUBSET WIZARD (SSW)'],
     ['Subsetter', 'SUBSETTER']
   ]
+
+  def kms_granule_data_formats
+    response = cmr_client.get_controlled_keywords('granule_data_format').body['short_name']
```

Comment:
take a look at app/concerns/controlled_keywords.rb. we should be checking for/handling a potential error response from the keywords endpoint